src/components/common/time-range-slider.js

```diff
@@ -65,7 +65,7 @@ export default class TimeRangeSlider extends Component {
       width: 288
     };
     this._animation = null;
-    this._sliderThrottle = throttle(this.props.onChange, 20);
+    this._sliderThrottle = throttle((value) => this.props.onChange(value), 20);
```

Comment:
Minor nitpick, might want to use the spread operator to apply all arguments to ensure there's no issues in the future when more arguments are used in the `onChange` prop method.

Reply:
done